python/ray/tune/examples/FCNet.py

```diff
@@ -0,0 +1,196 @@
+""" Code from https://github.com/automl/nas_benchmarks """
```

Comment:
We should jsut import this entirely from https://github.com/automl/nas_benchmarks right?